Config.in

```diff
@@ -0,0 +1,4 @@
+# Add project-specific packages for Buildroot here
+#
+# If these are non-proprietary, please consider contributing them back to
+# Nerves or Buildroot.
```

Comment:
I don't think that you meant to include `Config.in` in this commit.

Reply:
Damnit. :hourglass:

Reply:
I'm good with PR if `Config.in` is removed. There's probably an argument for checking it in, but for a different PR.

Reply:
Yeah this one gets me every time.